pages/api/content/[type].js

```diff
@@ -63,8 +65,8 @@ const createContent = async (req, res) => {
         type,
         {
           ...content,
-          groupId: req.query.groupId ? req.query.groupId : null,
-          groupType: req.query.groupType ? req.query.groupType : null,
+          ...(req.query.groupId ? { groupId: req.query.groupId } : {}),
```

Comment:
same here, groupId and groupType is `null` for contents that do not belong in a group